app/car.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+import math
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: float
+
+    def calculation_of_road_costs(
+            self,
+            location_customer: list,
+            location_shop: list,

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]